packages/custom-url/config/doctrine/CustomUrl/CustomUrlRoute.orm.xml

```diff
@@ -9,13 +9,23 @@
         </id>
 
         <field name="path" type="string" />
+        <field name="history" type="boolean">
+            <options>
+                <option name="default">0</option>
+            </options>
+        </field>
 
         <many-to-one field="customUrl" target-entity="Sulu\CustomUrl\Domain\Model\CustomUrlInterface" inversed-by="routes">
             <join-column name="customUrl" referenced-column-name="uuid" nullable="false" on-delete="cascade" />
          </many-to-one>
 
+        <many-to-one field="targetRoute" target-entity="Sulu\CustomUrl\Domain\Model\CustomUrlRouteInterface">
+            <join-column name="target_route_uuid" referenced-column-name="uuid" nullable="true" on-delete="set null" />
+        </many-to-one>
+
         <indexes>
-            <index name="routes" columns="path" />
+            <index name="idx_route_path" columns="path" />
+            <index name="idx_route_history" columns="history" />
```

Comment:
Index name need to be unique over the DB and shoould so have table name as a prefix.

```suggestion
            <index name="custom_url_route_path_idx" fields="path" />
            <index name="custom_url_route_history_idx" fields="history" />
```

similar to: https://github.com/sulu/sulu/blob/4ef044f0223ee0c814f295f87fb6e7ff5076a80c/packages/route/config/doctrine/Route/Route.orm.xml#L14